src/platform/mhv/secondary-nav/containers/MhvSecondaryNav.jsx

```diff
@@ -27,19 +39,27 @@ export const mhvSecNavItems = [
     iconClass: 'fas fa-prescription-bottle',
     href: `/my-health/medications`,
   },
-  {
-    title: 'Records',
-    iconClass: 'fas fa-file-medical',
-    href: `/my-health/medical-records`,
-  },
+  medicalRecordsLink,
 ];
 
 /**
  * MHV secondary navigation bar.
  * @returns the navigation bar
  */
 const MhvSecondaryNav = () => {
-  return <MhvSecondaryNavMenu items={mhvSecNavItems} />;
+  const items = [...mhvSecNavItems];
+  const { loading, mhvTransitionalMedicalRecordsLandingPage } = useSelector(
+    state => state.featureToggles,
+  );
+
+  if (loading) return <></>;
```

Comment:
I think this loading should go in the component since all the logic was added in there.

Reply:
The reason `loading` is here, is because we need to wait until the feature toggles are loaded in order to determine what links will be rendered.